src/vstest.console/Processors/TestAdapterLoadingStrategyArgumentProcessor.cs

```diff
@@ -211,13 +207,19 @@ private void ForceIsolation()
             return;
         }
 
+        EqtTrace.Warning(
+            $"{nameof(TestAdapterLoadingStrategyArgumentExecutor)}.{nameof(ForceIsolation)}: InIsolation setting is forced when {nameof(TestAdapterLoadingStrategy.Explicit)} strategy is used." +
+            "Test's will run in isolation."
```

Comment:
Assuming you meant plural form

```suggestion
            "Tests will run in isolation."
```